MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -299,6 +301,7 @@ public void onClick(View v) {
    */
   @UiThread
   public void onCreate(@Nullable Bundle savedInstanceState) {
+    created = true;
```

Comment:
No need to set to `false` in `onDestroy` I assume?

Reply:
correct, once an activity is destroyed it would create a new instance of MapView and created would be false at that point in time (as it's a new instance).